admin/spec/service/group.service.spec.js

```diff
@@ -71,6 +96,18 @@ describe('group.service', () => {
       expect(group).toEqual(groupMock)
       done()
     })
+
+    it('should return an error if schoolId or groupId are missing', async (done) => {
+      const groupId = '123456abcde'
+      const schoolId = null
+      try {
+        await groupService.getGroupById(groupId, schoolId)
```

Comment:
You need `fails` in *all* tests that are expecting a throw condition, to safe-guard against the error not being thrown.

E.g.
```
try {
   await service.doSomethingThatWeExpectToThrow(badArg)
   fail('expected to throw') // <------ this line
} catch (error) {
   expect(...)
}
```

Reply:
Unit tests have been updated in the parent PR.

Reply:
@pris54 i'm pretty confused by this old code/parent PR thing.  what's going on here?  are you managing 2 pieces of work? if so, how come we are adding this in one branch and modifying in another?

Reply:
@GuyHarwood This PR depends on PR 453 which I was expecting it would be merged long ago, but it hasn't hence I'm finding myself having to maintain this dependency.